examples/pi/Pi.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Exe</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.Spark.CSharp.Examples</RootNamespace>
-    <AssemblyName>Pi</AssemblyName>
+    <AssemblyName>SparkCLRPiExample</AssemblyName>
```

Comment:
The name of "SparkCLRPiExample" is a bit too long. Scala example is called "SparkPi", python example is called "Pi.py". If we prefer to stick with Scala notation, we can use SparkPi then (similar to sparkxml above)?